### PR TITLE
Add docs for Git protection rules

### PR DESCRIPTION
**Pull request by geofflamrock**
This PR adds docs for Git reference and Git resource rules to channels.
[sc-76671]

### CODE REVIEW

File: src/pages/docs/releases/channels/index.md [hunk 1]
```diff
@@ -40,15 +40,21 @@ As you add more channels, you'll notice that they are arranged in alphabetical o
 4. If you want to make this the default Channel for the project, click the **Default Channel** check-box.
 5. Design the [version rules](#version-rules) that will be used to enforce which versions of your packages are deployed to this channel.
 
-## Design the version rules {#version-rules}
+## Channel rules
 
-Version rules assist in selecting the correct versions of packages for the Channel.  They are only used when creating a release, either manually or via [project triggers](/docs/projects/project-triggers).
+Channels allow to you to configure rules to ensure that package versions and Git resources that meet specific criteria can be deployed using the channel.
+
+When creating a release for a channel with rules, an option can be configured on the project to allow the channel rules to be ignored. This option is disabled by default on new projects, but can be enabled using in project settings.
```
Comment (lourens-octopus) [inline src/pages/docs/releases/channels/index.md:47]: `but can be enabled using in project settings.`

Should this be enabled in project settings or using the project settings? Also do we tend to refer to project settings as a proper noun or is it the project's settings?

File: src/pages/docs/releases/channels/index.md [hunk 2]
```diff
@@ -83,6 +89,53 @@ The **Design Version Rule** window will show a list of the packages that will de
...
+![Git resource rules example](/docs/releases/channels/images/git-resource-rules.png)
+:::
+
+### Git reference rules (version controlled projects)
+
+:::div{.hint}
+Support for Git reference rules is currently rolling out to Octopus Cloud.
+:::
+
+Git reference rules assist in ensuring that only the correct branches and tags can be used as the Git reference for a release in the channel for version controlled projects. They are used when creating a release, either manually or via [project triggers](/docs/projects/project-triggers).
```
Comment (lourens-octopus) [inline src/pages/docs/releases/channels/index.md:115]: `the correct branches and tags`

Do you think configured would be more fitting here than correct?

File: src/pages/docs/releases/channels/index.md (hunk 1 above)
Comment (geofflamrock) [reply src/pages/docs/releases/channels/index.md:47]: Ah good pickup, that was a "writing docs at the end of the day" special 😁. I'll take a look into how we refer to it in other places and follow.

File: src/pages/docs/releases/channels/index.md (hunk 2 above)
Comment (geofflamrock) [reply src/pages/docs/releases/channels/index.md:115]: Yep that does sound better thanks

File: src/pages/docs/releases/channels/index.md (hunk 1 above)
Comment (geofflamrock) [reply src/pages/docs/releases/channels/index.md:47]: Looks like we "project settings" elsewhere so I'll go with that

**Review by lourens-octopus — APPROVED** [review]

**Comment by steve-fenton-octopus** [comment]
Bonus points for updating the `modDate` - thank yoU!

**Comment by steve-fenton-octopus** [comment]
Hi @geofflamrock - is this one waiting for a product release, or can it be merged now?

**Comment by geofflamrock** [comment]
> Hi @geofflamrock - is this one waiting for a product release, or can it be merged now?

@steve-fenton-octopus yep just waiting on rollout of the feature to commence to Octopus Cloud, I'll freshen up the PR and expect we'll merge it in the new couple of days or so.

**Review by steve-fenton-octopus — COMMENTED** [review]
LGTM!

**Review by lourens-octopus — APPROVED** [review]

**Pull request merged**